### PR TITLE
chore(presets): replace loading icon of ai generation

### PR DESCRIPTION
**Pull request by L-Sun**
Close AFF-991

### CODE REVIEW

**Comment by L-Sun** [comment]
* **#7031** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7031?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @L-Sun and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by fundon — APPROVED** [review]

File: packages/presets/src/ai/messages/_common/generating-placeholder.ts [hunk 1]
```diff
@@ -31,12 +35,16 @@ class AIGeneratingPlaceholder extends WithDisposable(LitElement) {
   height: number = 300;
 
   protected override render() {
+    const theme = getThemeMode();
```
Comment (fundon) [inline packages/presets/src/ai/messages/_common/generating-placeholder.ts:38]: We may need to listen to `mediaQuery` to get the latest theme in real time.

Comment (L-Sun) [reply packages/presets/src/ai/messages/_common/generating-placeholder.ts:38]: This is ok, because we can open settings when the ai-pannel opened

Comment (fundon) [reply packages/presets/src/ai/messages/_common/generating-placeholder.ts:38]: User may open the system settings.

**Pull request merged**